src/components/CallToAction.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { Link } from "gatsby";
+
+const CallToAction = () => (
+  <section>
+    <h1>Have you completed the pre-onboarding checklist?</h1>
```

Comment:
Let's change this to an `h3` tag. 

Let's also make this title a prop so that it's reusable. I'd call it a prop like "title" or something similar.